uploader.json

```diff
@@ -543,6 +543,11 @@
             "pattern": [],
             "symbolic_name": "lsblk_pairs"
         },
+        {
+            "command": "/bin/lscpu",
```

Comment:
@psachin this should be `/usr/bin/lscpu` to work on RHEL6.  Same with change below.